core/codenvy-hosted-workspace-activity/src/main/java/com/codenvy/activity/server/WorkspaceActivityService.java

```diff
@@ -56,9 +56,6 @@ public WorkspaceActivityService(WorkspaceActivityManager workspaceActivityManage
     @Path("/{wsId}")
     public void active(@PathParam("wsId") String wsId) throws ForbiddenException, NotFoundException, ServerException {
```

Comment:
because there is no javadoc as well on the method I see only
```
    @PUT
    @Path("/{wsId}")
    public void active(@PathParam("wsId") String wsId) throws ForbiddenException, NotFoundException, ServerException {
```